docs/Feature Documentation/Audio and Video Streaming/index.md

```diff
@@ -6,23 +6,9 @@ Before continuing, follow the [Install and Run Guide](../../getting-started/inst
 
 ### HMI Setup
 
-The Generic HMI does not currently support streaming.
+The [Generic HMI](https://github.com/smartdevicelink/generic_hmi) and [SDL HMI](https://github.com/smartdevicelink/sdl_hmi) both support streaming audio and some video formats in the browser using ffmpeg to transcode the video to VP8 WEBM. Instructions to install the required dependencies can be found in the HMI README, and you will need to run the backend server component (`./deploy_server.sh`) which handles the transcoding. Once you start a video stream it will take a few seconds for the transcoding session to begin and your video stream should appear in the browser within about 10 seconds.
```

Comment:
```suggestion
The [Generic HMI](https://github.com/smartdevicelink/generic_hmi) and [SDL HMI](https://github.com/smartdevicelink/sdl_hmi) both support streaming audio and some video formats in the browser using ffmpeg to transcode the video to VP8 WEBM. Instructions to install the required dependencies can be found in the HMI README 

- [SDL HMI Dependencies](https://github.com/smartdevicelink/sdl_hmi/tree/master#dependencies)
- [Generic HMI Dependencies](https://github.com/smartdevicelink/generic_hmi/tree/master#dependencies)

In the HMI directory, you will need to run the backend server component (`./deploy_server.sh`) which handles the transcoding. 

!!! NOTE
Once you start a video stream it will take a few seconds for the transcoding session to begin and your video stream should appear in the browser within about 10 seconds.
!!!
```
The dependencies links should work once we merge to master